src/docs/Components/Fields/previews/fields-state-mandatory.preview.html

```diff
@@ -0,0 +1,31 @@
+<div class="example">
+  <div class="mc-fields">
+    <label class="mc-fields__label" for="hover">Label<span class="mc-fields__mandatory">mandatory</span></label>
+    <input type="text" class="mc-text-input is-hover" id="hover" placeholder="Hover" name="example" />
+  </div>
+</div>
+<div class="example">
+  <div class="mc-fields">
+    <label class="mc-fields__label" for="focus">Label<span class="mc-fields__mandatory">mandatory</span></label>
+    <input type="text" class="mc-text-input is-focus" id="focus" placeholder="Focus" name="example" />
```

Comment:
If it's mandatory, you need the `required` and `aria-required` attributes on the input. check for this in all examples.